src/components/Header/Header.js

```diff
@@ -11,11 +15,24 @@ const Header = () => {
       }
     }
   `);
+  const { isOpened, handleTogglingOpenedStatus } = useToggleOpenedStatus();
+  const headerStatusClass = isOpened ? 'main-header--active' : '';
+  const burgerStatusClass = isOpened ? 'burger-menu--open' : 'burger-menu--closed';
   const email = 'mail@halo-lab.com';
+  const onBurgerMenuClick = () => {
+    handleTogglingOpenedStatus();
```

Comment:
handleTogglingOpenedStatus function does not change the state to a straight line, it does not need to be wrapped in onBurgerMenuClick

Reply:
Fixed.